### PR TITLE
[ENHANCMENT] PagerDuty: New or Updated incident - add more data in event body #5377

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/pagerduty/sources/common-webhook.mjs [hunk 1]
```diff
@@ -112,20 +112,24 @@ export default {
...
+    //     incident,
+    //   },
+    // ] = messages;
 
-    this.$emit(payload, this.getMetadata(payload));
+    // const payload = {
+    //   id,
+    //   event: messageEvent,
+    //   incident,
+    // };
```
Comment (andrewjschuang) [inline components/pagerduty/sources/common-webhook.mjs:131]: Can these commented lines be removed?

**Review by andrewjschuang — DISMISSED** [review]
LGTM!
I left just one comment below

File: components/pagerduty/sources/common-webhook.mjs (hunk 1 above)
Comment (lcaresia) [reply components/pagerduty/sources/common-webhook.mjs:131]: Yes, sorry!

**Comment by vunguyenhung** [comment]
Hello, I couldn't create account for this either. I have checked the code and it seems okay, so I will have this ready for release!

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**